mkdocs.yml

```diff
@@ -38,7 +38,6 @@ nav:
 - Bootstrapper: bootstrapper.md
 - Advanced usage: advanced-usage.md
 - Analyzer: /projects/analyzers
-- Snippets for FakeItEasy in SideWaffle: snippets-for-fakeiteasy-in-sidewaffle.md

```

Comment:
And remove it from the sln?

Reply:
I *knew* I had to be forgetting something...

Reply:
Amended